package.json

```diff
@@ -11,14 +11,14 @@
     "build": "turbo run build",
     "clean": "git clean -fdx -e '.env*'",
     "db": "bun run --filter @hebo/database db",
-    "dev": "turbo run dev --parallel",
+    "dev": "sh -c 'trap \"bun run postdev\" EXIT INT TERM; VITE_API_URL=http://localhost:3001 VITE_GATEWAY_URL=http://localhost:3002 turbo run dev --parallel'",
```

Comment:
Alternative suggestion to avoid duplicating URLs / ports in package.json:

1/ Detect isLocalDev based on whether proces.env.TURBO_HASH variable is set (and VITE_{API,GATEWAY}_URL are NOT set)
2/ Default ports in console/service.ts point to the same defaults as in API / Gateway
3/ Still allow manual override of URLs in console .env file via VITE_{API,GATEWAY}_URL